cmd/kubeadm/app/phases/addons/proxy/manifests.go

```diff
@@ -80,7 +80,9 @@ spec:
         - --config=/var/lib/kube-proxy/{{ .ProxyConfigMapKey }}
         - --hostname-override=$(NODE_NAME)
         securityContext:
-          privileged: true
+	  capabilities:
+	    add:
+	    - NET_ADMIN
```

Comment:
this seems to be failing with:

```
I1106 00:14:37.172875     175 round_trippers.go:444] POST https://kind-control-
yaml: line 30: found character that cannot start any token
unable to decode kube-proxy daemonset
k8s.io/kubernetes/cmd/kubeadm/app/phases/addons/proxy.createKubeProxyAddon
	cmd/kubeadm/app/phases/addons/proxy/proxy.go:145
k8s.io/kubernetes/cmd/kubeadm/app/phases/addons/proxy.EnsureProxyAddon
	cmd/kubeadm/app/phases/addons/proxy/proxy.go:58
k8s.io/kubernetes/cmd/kubeadm/app/cmd/phases/init.runKubeProxyAddon
	cmd/kubeadm/app/cmd/phases/init/addons.go:101
k8s.io/kubernetes/cmd/kubeadm/app/cmd/phases/workflow.(*Runner).Run.func1
	cmd/kubeadm/app/cmd/phases/workflow/runner.go:234
k8s.io/kubernetes/cmd/kubeadm/app/cmd/phases/workflow.(*Runner).visitAll
```

try:
```
securityContext:
      capabilities:
        add: ["NET_ADMIN"]
```

Reply:
Drat.  Ok.  I'll be able to try that in a couple hours